libr/include/r_core.h

```diff
@@ -116,6 +116,11 @@ typedef struct r_core_undo_t {
 	ut64 offset;
 } RCoreUndo;
 
+typedef enum {
```

Comment:
well if he deletes this it wont compile :D

Reply:
yep